Detectors/TPC/workflow/include/TPCWorkflow/TPCFactorizeIDCSpec.h

```diff
@@ -203,7 +214,7 @@ class TPCFactorizeIDCSpec : public o2::framework::Task
     auto it = std::find(mTFRangeIDCDelta.begin(), mTFRangeIDCDelta.end(), tf);
     if (it != mTFRangeIDCDelta.end()) {
       const int index = std::distance(mTFRangeIDCDelta.begin(), it);
-      mTimeStampRangeIDCDelta[index] = processing_helpers::getCreationTime(pc);
+      mTimeStampRangeIDCDelta[index] = processing_helpers::getTimeStamp(pc, getOrbitReset(pc)) / 1000;
```

Comment:
I guess the `getOrbitReset(pc)` can also go to `processing_helpers`, then one can simply have `getTimeStamp(pc)` and call the `getOrbitReset(pc)` from within.